instrument/methods.go

```diff
@@ -115,3 +115,45 @@ func NewMethodMetrics(scope tally.Scope, methodName string, samplingRate float64
 		SuccessLatency: newSampledTimer(scope.Timer(methodName+".success-latency"), samplingRate),
 	}
 }
+
+// BatchMethodMetrics is a bundle of common metrics for methods with batch semantics
+type BatchMethodMetrics struct {
+	RetryableErrors    tally.Counter
+	NonRetryableErrors tally.Counter
+	Success            tally.Counter
+	Latency            tally.Timer
+}
+
+// NewBatchMethodMetrics creates new batch method metrics
+func NewBatchMethodMetrics(
+	scope tally.Scope,
+	methodName string,
+	samplingRate float64,
+) BatchMethodMetrics {
+	return BatchMethodMetrics{
+		RetryableErrors:    scope.Counter(methodName + ".retryable-errors"),
```

Comment:
Perhaps also add just an ".errors" for backwards compatibility (and also just to get an indication of total errors too)?